### PR TITLE
Fixed few queries to be postgresql compatible 261

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

**Comment by mlassnig** [comment]
running test now against psql 9.6

**Comment by vingar** [comment]
should fail on 
DATA IDENTIFIERS (CORE): Delete dids ... ERROR

======================================================================
ERROR: DATA IDENTIFIERS (CORE): Delete dids
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/opt/rucio/lib/rucio/tests/test_did.py", line 59, in test_delete_dids
    delete_dids(dids=dsns, account='root')
  File "/opt/rucio/lib/rucio/db/sqla/session.py", line 349, in new_funct
    result = function(*args, **kwargs)
  File "/opt/rucio/lib/rucio/core/did.py", line 574, in delete_dids
    bindparam("deleted_at", datetime.utcnow())).\
  File "/usr/lib64/python2.7/site-packages/sqlalchemy/orm/scoping.py", line 157, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/sqlalchemy/orm/session.py", line 1362, in query
    return self._query_cls(entities, self, **kwargs)
  File "/usr/lib64/python2.7/site-packages/sqlalchemy/orm/query.py", line 139, in __init__
    self._set_entities(entities)
  File "/usr/lib64/python2.7/site-packages/sqlalchemy/orm/query.py", line 148, in _set_entities
    entity_wrapper(self, ent)
  File "/usr/lib64/python2.7/site-packages/sqlalchemy/orm/query.py", line 3978, in __init__
    self.use_id_for_hash = not type_.hashable
AttributeError: 'str' object has no attribute 'hashable'
-------------------- >> begin captured logging << --------------------
root: INFO: Removing did mock:dsn_fcb565d0f9024ccd94aef74f182587c3 (DATASET)
--------------------- >> end captured logging << ---------------------

**Comment by mlassnig** [comment]
confirmed for the hashable error, other than that looks fine

**Comment by mlassnig** [comment]
running testsuite now

**Comment by mlassnig** [comment]
can confirm it's ok now for psql

File: lib/rucio/core/did.py [hunk 1]
```diff
@@ -555,30 +555,32 @@ def delete_dids(dids, account, session=None):
...
-                              bindparam("deleted_at", datetime.utcnow())).\
-                filter(and_(models.DataIdentifierAssociation.scope == did['scope'],
-                            models.DataIdentifierAssociation.name == did['name']))
-            ins = Insert(table=models.DataIdentifierAssociationHistory, inline=True).\
-                from_select(('scope', 'name', 'child_scope', 'child_name', 'did_type',
-                             'child_type', 'bytes', 'adler32', 'md5', 'guid', 'events',
-                             'rule_evaluation', 'did_created_at', 'created_at', 'updated_at',
-                             'deleted_at'), q)
-            session.execute(ins)
+            # Disable for postgres
```
Comment (mlassnig) [inline lib/rucio/core/did.py:558]: this is fine for now. we need to fix this in a later patch.

**Review by mlassnig — APPROVED** [review]

**Pull request merged**